qCC/ccCommandLineCommands.cpp

```diff
@@ -730,12 +730,31 @@ bool CommandLoadCommandFile::process(ccCommandLineInterface& cmd)
 			}
 
 			//inject back all the arguments to the cmd.arguments()
-			while (!processedArgs.isEmpty()) {
+			bool commentWholeLine = false;
+			while (!processedArgs.isEmpty())
+			{
 				QString processedArg = processedArgs.takeFirst();
-				if (!processedArg.isEmpty()) {
-					cmd.print(QObject::tr("\t[%1] %2").arg(insertingIndex).arg(processedArg));
-					cmd.arguments().insert(insertingIndex, processedArg);
-					insertingIndex++;
+				if (!processedArg.isEmpty())
+				{
+					bool commentArgument = false;
+					QString commentStr = "[COMMENT] ";
```

Comment:
I wouldn't initialize this string each time, as creating a QString object is quite expensive... Let me change this and you'll tell me if it's ok with you.